solutions/logging/core-experimentation/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
```

Comment:
since this is a new package, an entry will need to be added in [release-please-config.json](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/release-please-config.json)

Reply:
Fixed, please resolve the comment if satisfied.